app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    storage_cache = {}
+
+    def wrapper(*args: int) -> int:
```

Comment:
The return type hint `-> int` for the `wrapper` function is incorrect. Since the `wrapper` function returns the result of `func(*args)`, its return type should match the return type of `func`. Consider using `-> Any` or the return type of `func` if known.